wiki/mapping/glossary.md

```diff
@@ -16,232 +16,242 @@ let us know in #mapping-discussion!
-| **Staggers** | Sliders placed with spacing too large for the player to hit in one motion, considered a mismap. Occurs when sliders are placed lower than 1/8 precision. <details><summary>**Example Image**</summary>![Picture of stagger](~@images/mapping/stagger.jpg)</details>|
-| **Staircase** | A pattern where a block's direction points towards the following block. Does not include blocks on the same beat. Can be a potential hitbox issue where a block is placed in the postswing of a previous block. <details><summary>**Example Image**</summary>![Picture of staircase](~@images/mapping/staircase.jpg)</details>|
-| **Stream** | A steady, sustained pattern of notes, typically at 1/4 precision. <details><summary>**Example Image**</summary>![Picture of stream](~@images/mapping/stream.png)</details>|
-| **Strobe** | 1) n. A rapidly flashing on/off light <br> 2) v. To cause a light to flash on/off or on/flash when lighting a map. |
-| **Swings per second** | A variation on [Notes per second](#n) that isn't inflated by stacks of sliders. Can give a better representation of a map's difficulty. |
+| **Scissor** | When a red and blue [Note](#n) are on the same timing, and are hit simultaneously in opposite directions. Also known as [Cucumber](#c). See also: [Crossover Scissor](#c). <details><summary>**Example Image**</summary>![Picture of scissor](~@images/mapping/scissor-cucumber_example.png)</details> |
+| **Scoop** | A [Pattern](#p) of [Blocks](#b) where the player makes a "scooping" motion to play the pattern. Typically a left or right note followed by an up note in the bottom row. Give the player an ample amount of time to react when using scoops - a half to full beat is recommended. <details><summary>**Example Image**</summary>![Picture of scoop](~@images/mapping/scoop_example.png)</details> |
+| **ScoreSaber** | The organization responsible for managing the ranked Beat Saber leaderboards. The Beat Saber leaderboards website can be found [here](https://scoresaber.com/). See also: [Performance Points](#p). |
+| **Setup** | The process of maneuvering the player's arms into position for a [Pattern](#p) via appropriate placement of the preceding [Blocks](#b). |
+| **Single** | A single [Block](#b) hit with one saber. Typically makes up the majority of [Patterns](#p) in a map. See also: [Double](#d). |
+| **Single BPM** | A song with a consistent [BPM](#b) from start to finish with no variation. Also known as [Fixed BPM](#f). See also: [Variable BPM](#uv). |
+| **Sliders** | A series of same-colored [Dot Blocks](#d) or [Arrow Blocks](#a) spaced close enough together for the player to sweep the saber through in one motion. Placed at a [Precision](#p) of 1/12 or faster. <details><summary>**Example Image**</summary>![Picture of sliders](~@images/mapping/sliders.png)</details> |
+| **SongCore** | A mod developed by Kyle1314 for handling custom song additions in Beat Saber. |
```

Comment:
Got the wrong numbers here. It's Kyle1413.

Reply:
Good catch! I copied the name from the Mapping Extensions entry... which also got it wrong, heh. I'll fix both.